pushover-notifications-edd-ext.php

```diff
@@ -361,7 +361,7 @@ public function execute_daily_discount_notices() {
 			return;
 
 		$args = array(
-			'post_status' => 'active'
+			'status' => 'active',
```

Comment:
I believe this does need to be `post_status` for EDD 2.10 though.

Honestly I'm wondering if it would work to be lazy and set the array up like:

```php
$args = array(
    'status'      => 'active',
    'post_status' => 'active',
);
```
It seems like the incorrect parameter would just be ignored/discarded by EDD?

Reply:
Blargh you're right -- I was thinking of payment queries, where `status` is okay.

Reply:
```suggestion
			'status'         => 'active',
			'post_status' => 'active',
```

Making this an official suggestion, as I've tested manually in both 2.11 and 3.0. I think we could merge this once this is in and then deal with the additional issue Chris raised in a new PR.